adding etherpad lite installation manual

ezzra opened the pull request:
first try to contribute with an installation manual

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+.. note:: You need to keep Etherpad Lite updated by pulling the current sourcecode from Github with ``git pull origin``. Afterwards, use ``supervisorctl restart etherpad`` to restart with the new codebase.
+
+
+.. _Loremipsum: https://en.wikipedia.org/wiki/Lorem_ipsum
```
That's a placeholder ☺️

nichtmax replied:
I found a feed though: https://github.com/ether/etherpad-lite/releases.atom

So maybe add a note with the correct feed above?

```
.. note:: Check the update feed_ regularly to stay informed about the newest version.
```

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+.. _MySQL: https://manual.uberspace.de/en/database-mysql.html
+.. _supervisord: https://manual.uberspace.de/en/daemons-supervisord.html
+.. _domains: https://manual.uberspace.de/en/web-domains.html
+.. _feed: https://github.com/lorem/ipsum/releases.atom
```
That's a placeholder ☺️

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+Updates
+=======
+
+.. note:: You need to keep Etherpad Lite updated by pulling the current sourcecode from Github with ``git pull origin``. Afterwards, use ``supervisorctl restart etherpad`` to restart with the new codebase.
```
No need for a "note" there I think. Take a look at the rendered [ghost guide](https://master.lab.uberspace.de/en/ghost.html) to see what that looks like.

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
```
What's the output of that? What should the user expect? I suggest changing to:

```
[etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
Cloning into '/home/etherpad/etherpad'...
remote: Counting objects: 29789, done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 29789 (delta 9), reused 16 (delta 7), pack-reused 29768
Receiving objects: 100% (29789/29789), 19.25 MiB | 20.58 MiB/s, done.
Resolving deltas: 100% (21251/21251), done.
[etherpad@stardust ~]$ 
```

This way it's totally clear to everybody if it worked or not.

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ ~/etherpad/bin/installDeps.sh
```
Same here:

```
[etherpad@stardust ~]$ ~/etherpad/bin/installDeps.sh
Copy the settings template to settings.json...
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
[...]
Clearing minified cache...
Ensure custom css/js files are created...
[etherpad@stardust ~]$ 
```

You can always shorten unimportant output with `[...]` like I did here.

nichtmax commented on source/etherpad.rst:
```diff
@@ -0,0 +1,194 @@
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ mysql -e "CREATE DATABASE $USER_etherpad COLLATE utf8mb4_unicode_ci;"
```
Doesn't work for me:

```
[moritz@stardust ~]$ mysql -e "CREATE DATABASE $USER_etherpad COLLATE utf8mb4_unicode_ci;"
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'COLLATE utf8mb4_unicode_ci' at line 1
```

luto replied:
`CREATE DATABASE ${USER}_etherpad COLLATE utf8mb4_unicode_ci;`. Otherwise bash tries to evaluate `$USER_etherpad`.

ezzra commented:
Ok, I addressed all the problems and fixed them. 
- I also changed the git clone, because the standard branch on etherpad is a developer branch. 
- I also added a logo, but that one is grabbed from the etherpad.org page, so maybe its better without logo.
- I had problems, building the html with the code-block:: json, but maybe its working for you?

Because the changes are splitted now in two commits and your older comments are buried now...I think I should make a rebase and completely new pull request, what do you think?

Thanks for the suggestions! Feel free to add more :)

ezzra commented:
I will adapt the guide to the new requirements (username, includes, ..) and start a new pull request

luto commented:
@ezzra yea, sorry :/ we're quite early in the evolution of the lab, so things still shift around quite q bit. Thanks for your will to rework your guide <3

ezzra commented:
no matter, I also think the new requirements set a better base for contributions ;)
